### DISCUSSION TITLE
Alcantara2 - Acova: MQTT message published incorrect?

### DISCUSSION THREAD

**Discussion by vfd7** [Koenkk/zigbee2mqtt#10866] [Devices]
Hello,

I have an acova heater called alcantara2 that I want to make it work with Z2Q.
I have used acova.js and modified it slighty to support this new acova heater  (config: climate).
Another heater from acova is defined (percale) which is from the same family so I had just to udpate the model type.

I check the MQTT message pubhlished and something is puzlling me. 
Here is the message that is published in the zigbee2mqtt server console:

MQTT publish: topic 'zigbee2mqtt/Chauffage Bureau', payload '{"linkquality":0,"local_temperature":21,"occupied_heating_setpoint":19,"running_state":"idle","system_mode":{"system_mode":"off"},"unoccupied_heating_setpoint":15.5}'

I am new to the MQTT protocol so I have a question related to "system_mode":{"system_mode":"off"}.
I would have expected "system_mode":"off"
Is my understanding correct?
I think that this is not related to acova.js contnt but I am still not 100% sure tough. 

Could you help and clarify if my understanding is correct?
Thank you.
vfd.

**Comment by milescherif** [comment]
Hi,

Is your Acova alcantara2 heater paired on the interface ?
I tried to pair it several times (mode and + button) but nothing is found on the interface.
[Here the steps that I followed](https://www.zigbee2mqtt.io/devices/ALCANTARA2.html).
I'm using a conbee2.

I think that the js file is used to configure the heater after been paired. And I'm stuck on the first step.

Thanks for your help
